Update webhook docs (#5994)
PostHog/posthog.com: contents/docs/webhooks/slack.md

showTitle: true
---

> You can receive messages in Slack whenever any of your specified [actions](/docs/user-guides/actions) are triggered. 

Reviewer: ```suggestion
You can receive messages in Slack whenever your [actions](/docs/user-guides/actions) trigger. 
```

> **Note:** Currently it's only possible to receive messages in a single Slack channel. If you'd like to receive messages in more than one channel, you can use the [Zapier Connector](/apps/zapier-connector).

> ## How to setup the Slack webhook

Reviewer: Title and h2s

### 1. Create an app in Slack

> Go to the [Slack developer dashboard]((https://api.slack.com/apps?new_app=1)) and create a new app. Call it "PostHog" and connect it to the workspace of your choice.

Reviewer: ```suggestion
Go to the [Slack developer dashboard](https://api.slack.com/apps?new_app=1) and create a new app. Call it "PostHog" and connect it to the workspace of your choice.
```

Optional: Feel free to use an image from [here](/media) as the app's logo.

![image](https://user-images.githubusercontent.com/53387/78574619-86939580-782a-11ea-8617-caf1ffe2783a.png)

### 2. Create a webhook in Slack

> Go to the 'Incoming Webhooks' page for your newly-created app and toggle 'Activate Incoming Webhooks' to turn it on. Then click on 'Add New Webhook to Workspace' and select the channel that the notifications will be posted to.

Reviewer: ```suggestion
Go to the 'Incoming Webhooks' page for your newly-created app and toggle 'Activate Incoming Webhooks' to turn it on. Then click on 'Add New Webhook to Workspace' and select the channel that the action posts to.
```

![image](https://user-images.githubusercontent.com/53387/78574881-ec801d00-782a-11ea-9b87-8a40e49dd912.png)

### 3. Connect your webhook to PostHog

> Copy the Webhook URL from Slack, navigate to project settings in PostHog paste the url into the "Webhook integration section".

Reviewer: ```suggestion
Copy the Webhook URL from Slack, navigate to [project settings](https://app.posthog.com/project/settings) in PostHog, and paste the URL into the "Webhook integration section".
```
